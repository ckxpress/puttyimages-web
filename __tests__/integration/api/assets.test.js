const api = require('./api');

test('get asset info', async () => {
  const response = await api.get(
    'assets/04f01eb90f7be1b1628218c5a9f1f87c5704bc79548f15673695a3e7264ba88b'
  );
  expect(response.data).toMatchSnapshot();
});
