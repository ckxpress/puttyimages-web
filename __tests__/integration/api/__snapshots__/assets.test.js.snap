// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get asset info 1`] = `
Object {
  "createdAt": "2018-06-20T00:00:00.000Z",
  "description": "avocado",
  "fingerprint": "04f01eb90f7be1b1628218c5a9f1f87c5704bc79548f15673695a3e7264ba88b",
  "height": 15,
  "ipfs": "QmfEcrSyPS3K7WNRZLseGu3TqSpQqRToJsvT4svUe1UDmd",
  "ipld": "zdpuAqoWEAALng3FeUcF8GY66UsK17wd3fGxr9XbXu6baoKLN",
  "license": "cc0",
  "like": Object {
    "count": 0,
  },
  "tags": Array [
    Object {
      "name": "avocado",
    },
    Object {
      "name": "fruit",
    },
    Object {
      "name": "berry",
    },
  ],
  "updatedAt": "2018-06-20T00:00:00.000Z",
  "url": "http://localhost:8080/ipfs/QmfEcrSyPS3K7WNRZLseGu3TqSpQqRToJsvT4svUe1UDmd",
  "user": Object {
    "displayName": "Alexandre",
    "likecoinId": "likecoin_id_1",
  },
  "wallet": "address 1",
  "width": 20,
}
`;
